apps/nativescript-demo-ng/src/polyfills.ts

```diff
@@ -6,7 +6,9 @@
 import '@nativescript/core/globals';
 // Install @nativescript/angular specific polyfills
 import '@nativescript/angular/polyfills';
-
+import { AbortController, AbortSignal } from '@nativescript/core/abortcontroller';
+(global as any).AbortController = AbortController;
+(global as any).AbortSignal = AbortSignal;
```

Comment:
should be in the pre-zone polyfills

Reply:
fixed but I'll leave comment in case want to adjust further - they are in pre-zone polyfills now.